### ISSUE TITLE
Is Numba necessary for Transducer?

### ISSUE THREAD

**Issue by KacperKubara** [speechbrain/speechbrain#1320]
Hi, 
I've seen that you use Numba for the implementation forward-backward algorithm. I was wondering what is the benefit of using Numba instead of trying to rely on PyTorch functionality to have it implemented? I personally feel like Numba is quite a heavy dependency in the Speechbrain package and it is sometimes troublesome to make it work with Cuda-enabled GPU.

https://github.com/speechbrain/speechbrain/blob/develop/speechbrain/nnet/loss/transducer_loss.py

**Comment by KacperKubara** [comment]
Hi, thanks for the answer. So I understand correctly, the new torchaudio implementation performance is good? Is this torchaudio implementation also part of Speechbrain's transducer, or do I need to import it by myself?

**Comment by TParcollet** [comment]
It is good, yes, but I don't know if we have an example yes. Basically, the idea is that you simply replace in the yaml our loss by the torchaudio.blabla.loss and give it the correct arguments. Maybe you will have to check in the train.py if few changes are necessary (what is given in parameters ect). @aheba Do we have an example of using the torchaudio RNNT loss?

**Comment by KacperKubara** [comment]
thanks. I found out that there is an easy way to use touchaudio functionality. There is an option to set it in yaml config file:

``` yaml
transducer_cost: !name:speechbrain.nnet.losses.transducer_loss
   blank_index: !ref <blank_index>
   use_torchaudio: True
```
